profiles/killbill/src/test/resources/org/killbill/billing/server/SpyCarBasic.xml

```diff
@@ -110,6 +111,7 @@
         </plan>
         <plan name="sports-monthly">
             <product>Sports</product>
+            <recurringBillingMode>IN_ARREAR</recurringBillingMode>
```

Comment:
Let's wait for CI, I suspect this will break some tests (the plan was `IN_ADVANCE` before).